definitions/variable/macro-economy/tag_population_deciles.yaml

```
@@ -0,0 +1,21 @@
+- Population Deciles:
+    - D1:
+        description: first Decile
```

Please use consistent lower-case here
```suggestion
        description: first decile
```